tests/config/pytest_modules.yml

```diff
@@ -1,2525 +1,2529 @@
 abacas:
-  - modules/abacas/**
-  - tests/modules/abacas/**
+- modules/abacas/**
+- tests/modules/abacas/**
```

Comment:
Please be careful about making spurious whitespace changes to files. It's good practice to use `git diff` to check for this kind of thing and then do your best to avoid making massive edits like this.

In this case the whitespace changes make the YAML linting CI tests with [Prettier](https://prettier.io/) fail, so it's a double problem.